src/main/java/org/jenkinsci/plugins/workflow/support/concurrent/ChainingListenableFuture.java

```diff
@@ -208,7 +208,7 @@ public void run() {
               ChainingListenableFuture.this.outputFuture = null;
             }
           }
-        }, Futures.newExecutorService());
+        }, MoreExecutors.directExecutor());
```

Comment:
Undoing a change made in #114 in favor of the upstream version. We should generally avoid changing vendored upstream code.